build-tools/tasks/styles.js

```diff
@@ -34,18 +36,18 @@ function compileStyleDictionary() {
   });
 }
 
-const rootDir = join(__dirname, '../../');
-
 function stylesTask(theme) {
-  return task(`styles:${theme.name}`, () => {
+  return task(`styles:${theme.name}`, async () => {
     const designTokensOutputDir = join(workspace.targetPath, theme.designTokensDir);
-    const primary = getTheme(theme.primaryThemePath);
-    const secondary = theme.secondaryThemePaths ? theme.secondaryThemePaths.map(path => getTheme(path)) : [];
-    const styleDictionaryName = basename(theme.primaryThemePath);
+    // eslint-disable-next-line no-unsanitized/method
```

Comment:
Eslint rule does not like dynamic code inside `import()` calls. 

I did not find a way to disable `import` calls validation but keep all others, so here go individual ignore comments